pex/interpreter.py

```diff
@@ -535,7 +535,6 @@ def _find_pyvenv_cfg(cls, maybe_venv_python_binary):
     @classmethod
     def _resolve_pyvenv_canonical_python_binary(
         cls,
-        real_binary,  # type: str
```

Comment:
Not relevant to the fix, but noted in the investigation as un-used; so I simplified for sanity sake.